src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -219,6 +213,45 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     }
   }
 
+  submitting(): boolean {
+    return this.isSubmitting;
+  }
+
+  getCaseId(): String {
+    return (this.caseEdit.caseDetails ? this.caseEdit.caseDetails.case_id : '');
+  }
+
+  getCancelText(): String {
```

Comment:
return type is JS String type, should be lowercase for Typescript string typing.